src/aind_metadata_mapper/bergamo/session.py

```diff
@@ -0,0 +1,548 @@
+                            PhotoStimulationGroup(
+                                group_index=0,
+                                number_of_neurons=int(
+                                    np.array(
+                                        photostim_groups[0]["rois"][1][
+                                            "scanfields"
+                                        ]["slmPattern"]
+                                    ).shape[0]
+                                ),
+                                stimulation_laser_power=int(
+                                    photostim_groups[0]["rois"][1][
+                                        "scanfields"
+                                    ]["powers"]
+                                ),
+                                number_trials=5,
+                                number_spirals=int(
+                                    photostim_groups[0]["rois"][1][
+                                        "scanfields"
+                                    ]["repetitions"]
+                                ),
+                                spiral_duration=photostim_groups[0]["rois"][1][
+                                    "scanfields"
+                                ]["duration"],
+                                inter_spiral_interval=photostim_groups[0][
+                                    "rois"
+                                ][2]["scanfields"]["duration"],
+                            ),
+                            PhotoStimulationGroup(
+                                group_index=0,
+                                number_of_neurons=int(
+                                    np.array(
+                                        photostim_groups[0]["rois"][1][
+                                            "scanfields"
+                                        ]["slmPattern"]
+                                    ).shape[0]
+                                ),
+                                stimulation_laser_power=int(
+                                    photostim_groups[0]["rois"][1][
+                                        "scanfields"
+                                    ]["powers"]
+                                ),
+                                number_trials=5,
+                                number_spirals=int(
+                                    photostim_groups[0]["rois"][1][
+                                        "scanfields"
+                                    ]["repetitions"]
+                                ),
+                                spiral_duration=photostim_groups[0]["rois"][1][
+                                    "scanfields"
+                                ]["duration"],
+                                inter_spiral_interval=photostim_groups[0][
+                                    "rois"
+                                ][2]["scanfields"]["duration"],
+                            ),
+                        ],
+                        inter_trial_interval=10,
```

Comment:
ditto

Reply:
I put all the hard-coded values into a settings class with defaults. Is this a good enough place to a pin on this project and open new tickets if we want to extract more information from the tif directory?